Makefile

```diff
@@ -47,18 +53,14 @@ deploy:
 documents/%.sts.html: documents/%.sts.xml saxon.jar
 	java -jar saxon.jar -s:$< -xsl:$(STS2HTMLXSL) -o:$@
 
-documents/%.sts.xml: documents/%.mn.xml target/$(JAR_FILE) | documents
+documents/%.sts.xml: documents/%.mn.xml | target/$(JAR_FILE) documents
```

Comment:
The portion before `|` means if the file needs to exist. The portion after `|` means run the target regardless if it exists. Why do we need to change here?